File: angular/src/app/add-module/location-dialog/domain/section.ts
import { LocationChicklets } from './chicklets';

export class LocationSection {
    constructor(private sectionId: string,
        private userId: string,
        private operationType: string,
        private chicklets: LocationChicklets[]) {
        this.sectionId = sectionId;
        this.userId = userId;
        this.operationType = operationType;
        this.chicklets = chicklets;
    }
}

File: InterestService/src/main/java/com/stackroute/interestservice/domain/Section.java
package com.stackroute.interestservice.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;


/*
 *  Section class is input format which comes from
 *  kafka education topic.
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Section {
    private String sectionId;
    private String userId;
    private String operationType;
    private Chicklets chicklets[];
}
